app/views/merge_requests/logs_outcomes.html.erb

```diff
@@ -0,0 +1,27 @@
+<% content_for :before_content do %>
+  <% title = "Users" %>
+  <% content_for :title, title %>
+  <%= govuk_back_link href: merge_request_path(@merge_request) %>
+<% end %>
+
+<h1 class="govuk-heading-l">
+  <span class="govuk-caption-l"><%= @merge_request.absorbing_organisation_name %></span>
+  Logs
+</h1>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% unless @merge_request.status == "request_merged" || @merge_request.status == "processing" %>
+      <h2 class="govuk-heading-m"><%= total_lettings_logs_after_merge_text(@merge_request) %></h2>
+      <p class="govuk-body">
+        <%= merging_organisations_lettings_logs_outcomes_text(@merge_request) %>
+      </p>
+
+      <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--m govuk-!-margin-top-8 govuk-!-margin-bottom-8 app-tab__underline">
```

Comment:
Could the breakline here and in relationship outcomes be the same?
Like this:
`<hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible govuk-!-margin-bottom-7 govuk-!-width-three-quarters ">`

The paragraph above adds a margin-bottom which together with margin-top-8 here leaves a larger gap above than below.

I'll edit the relationship outcomes page to 1) use the code above 2) wrap the content within `<div class="govuk-grid-row"> <div class="govuk-grid-column-two-thirds">` as you have here.

Reply:
https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/2619